public/tippy.umd.js.map

```diff
@@ -0,0 +1 @@
IAAd;EAAoBG,UAAAA,KAAK,EAALA,KAApB;EAA2ByW,UAAAA,KAAK,EAALA,KAA3B;EAAkCC,UAAAA,MAAM,EAANA;EAAlC,SAAP;EACD;;EACD,SAAK,MAAL;EACA,SAAK,OAAL;EAAc;EACZ,YAAMoB,OAAO,GAAGvC,IAAI,CAACwC,GAAL,OAAAxC,IAAI,EAAQmC,WAAW,CAAC9L,GAAZ,CAAgB,UAAC0L,KAAD;EAAA,iBAAWA,KAAK,CAACzX,IAAjB;EAAA,SAAhB,CAAR,CAApB;EACA,YAAMmY,QAAQ,GAAGzC,IAAI,CAAC0C,GAAL,OAAA1C,IAAI,EAAQmC,WAAW,CAAC9L,GAAZ,CAAgB,UAAC0L,KAAD;EAAA,iBAAWA,KAAK,CAACtX,KAAjB;EAAA,SAAhB,CAAR,CAArB;EACA,YAAMkY,YAAY,GAAGR,WAAW,CAAC1b,MAAZ,CAAmB,UAACsa,IAAD;EAAA,iBACtCkB,oBAAoB,KAAK,MAAzB,GACIlB,IAAI,CAACzW,IAAL,KAAciY,OADlB,GAEIxB,IAAI,CAACtW,KAAL,KAAegY,QAHmB;EAAA,SAAnB,CAArB;EAMA,YAAMxY,IAAG,GAAG0Y,YAAY,CAAC,CAAD,CAAZ,CAAgB1Y,GAA5B;EACA,YAAMG,OAAM,GAAGuY,YAAY,CAACA,YAAY,CAAChQ,MAAb,GAAsB,CAAvB,CAAZ,CAAsCvI,MAArD;EACA,YAAME,KAAI,GAAGiY,OAAb;EACA,YAAM9X,MAAK,GAAGgY,QAAd;;EACA,YAAMvB,MAAK,GAAGzW,MAAK,GAAGH,KAAtB;;EACA,YAAM6W,OAAM,GAAG/W,OAAM,GAAGH,IAAxB;;EAEA,eAAO;EAACA,UAAAA,GAAG,EAAHA,IAAD;EAAMG,UAAAA,MAAM,EAANA,OAAN;EAAcE,UAAAA,IAAI,EAAJA,KAAd;EAAoBG,UAAAA,KAAK,EAALA,MAApB;EAA2ByW,UAAAA,KAAK,EAALA,MAA3B;EAAkCC,UAAAA,MAAM,EAANA;EAAlC,SAAP;EACD;;EACD;EAAS;EACP,eAAOe,YAAP;EACD;EArCH;EAuCD;;EC9KD,IAAMrT,MAAc,GAAG;EACrBiD,EAAAA,IAAI,EAAE,QADe;EAErB7M,EAAAA,YAAY,EAAE,KAFO;EAGrBY,EAAAA,EAHqB,cAGlByG,QAHkB,EAGR;EACX,QAAOnE,SAAP,GAA4BmE,QAA5B,CAAOnE,SAAP;EAAA,QAAkBkL,MAAlB,GAA4B/G,QAA5B,CAAkB+G,MAAlB;;EAEA,aAASuP,YAAT,GAA2D;EACzD,aAAOtW,QAAQ,CAACgJ,cAAT,GACHhJ,QAAQ,CAACgJ,cAAT,CAAwB1M,KAAxB,CAA8BgT,QAA9B,CAAuCzT,SADpC,GAEHA,SAFJ;EAGD;;EAED,aAAS0a,WAAT,CAAqB9d,KAArB,EAA6D;EAC3D,aAAOuH,QAAQ,CAAC5C,KAAT,CAAemF,MAAf,KAA0B,IAA1B,IAAkCvC,QAAQ,CAAC5C,KAAT,CAAemF,MAAf,KAA0B9J,KAAnE;EACD;;EAED,QAAI+d,WAA8B,GAAG,IAArC;EACA,QAAIC,WAA8B,GAAG,IAArC;;EAEA,aAASC,cAAT,GAAgC;EAC9B,UAAMC,cAAc,GAAGJ,WAAW,CAAC,WAAD,CAAX,GACnBD,YAAY,GAAGlJ,qBAAf,EADmB,GAEnB,IAFJ;EAGA,UAAMwJ,cAAc,GAAGL,WAAW,CAAC,QAAD,CAAX,GACnBxP,MAAM,CAACqG,qBAAP,EADmB,GAEnB,IAFJ;;EAIA,UACGuJ,cAAc,IAAIE,iBAAiB,CAACL,WAAD,EAAcG,cAAd,CAApC,IACCC,cAAc,IAAIC,iBAAiB,CAACJ,WAAD,EAAcG,cAAd,CAFtC,EAGE;EACA,YAAI5W,QAAQ,CAACgJ,cAAb,EAA6B;EAC3BhJ,UAAAA,QAAQ,CAACgJ,cAAT,CAAwB8N,MAAxB;EACD;EACF;;EAEDN,MAAAA,WAAW,GAAGG,cAAd;EACAF,MAAAA,WAAW,GAAGG,cAAd;;EAEA,UAAI5W,QAAQ,CAAC1D,KAAT,CAAe6M,SAAnB,EAA8B;EAC5BqF,QAAAA,qBAAqB,CAACkI,cAAD,CAArB;EACD;EACF;;EAED,WAAO;EACLtS,MAAAA,OADK,qBACW;EACd,YAAIpE,QAAQ,CAAC5C,KAAT,CAAemF,MAAnB,EAA2B;EACzBmU,UAAAA,cAAc;EACf;EACF;EALI,KAAP;EAOD;EAnDoB,CAAvB;AAsDA;EAEA,SAASG,iBAAT,CACEE,KADF,EAEEC,KAFF,EAGW;EACT,MAAID,KAAK,IAAIC,KAAb,EAAoB;EAClB,WACED,KAAK,CAACpZ,GAAN,KAAcqZ,KAAK,CAACrZ,GAApB,IACAoZ,KAAK,CAAC5Y,KAAN,KAAgB6Y,KAAK,CAAC7Y,KADtB,IAEA4Y,KAAK,CAACjZ,MAAN,KAAiBkZ,KAAK,CAAClZ,MAFvB,IAGAiZ,KAAK,CAAC/Y,IAAN,KAAegZ,KAAK,CAAChZ,IAJvB;EAMD;;EAED,SAAO,IAAP;EACD;;EC/DDoR,KAAK,CAAClK,eAAN,CAAsB;EACpBR,EAAAA,OAAO,EAAE,CAACtC,WAAD,EAAcC,YAAd,EAA4BC,iBAA5B,EAA+CC,MAA/C,CADW;EAEpBqC,EAAAA,MAAM,EAANA;EAFoB,CAAtB;EAKAwK,KAAK,CAACoB,eAAN,GAAwBA,eAAxB;EACApB,KAAK,CAACgD,QAAN,GAAiBA,QAAjB;EACAhD,KAAK,CAACM,OAAN,GAAgBA,OAAhB;EACAN,KAAK,CAAC6H,UAAN,GAAmBzf,WAAnB;;;;;;;;"}
```

Comment:
Should this be empty?

Reply:
Why?

Reply:
Oh nevermind, it looks empty in the Github diff, but I see the full sourcemap in the raw view.